libraries/win32_helper.rb

```diff
@@ -0,0 +1,40 @@
+#
+# Author:: Piyush Awasthi (<piyush.awasthi@chef.io>)
+# Cookbook:: windows
+# Library:: win32_certstore_helper
+#
+# Copyright:: 2011-2018, Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'win32-certstore' 
+require "openssl"
+
+module Win32
```

Comment:
This whole helper should just get rolled into the certificates resource. As it stands right now anyone with this cookbook on their run list will end up loading win32-certstore and openssl which is going to slow down every chef run regardless of use. Also if / when we move this resource into chef itself it will be a lot easier if it's self contained.